gui/resources.xml

```diff
@@ -1104,6 +1141,7 @@ Vocabulary
           <property name="icon-name">mypaint-info-symbolic</property>
           <signal name="activate" handler="show_online_help_cb"/>
         </object>
+        <accelerator key="F1" modifiers="GDK_SHIFT_MASK"/>
```

Comment:
Ctrl- variants sound right for the scrapbook functions.

Sadly, there's not really any standard for media-function keys on laptops we could leverage (main laptop :rewind:=Fn+F8 :play_or_pause_button:=Fn+F9, :fast_forward:=Fn+F10, windows clunker uses F9/F10/F11 for the same)

It frees up F2 for "rename current layer", matching GIMP :smiley: